runtime/drive_handler.go

```diff
@@ -125,10 +125,20 @@ func (h *stubDriveHandler) createStubDrive(driveID, path string) error {
 	return nil
 }
 
+func filterOnlyStubDrives(drives []models.Drive) []models.Drive {
+	stubDrives := make([]models.Drive, 0)
+	for _, d := range drives {
+		if !*d.IsRootDevice {
+			stubDrives = append(stubDrives, d)
+		}
+	}
+	return stubDrives
+}
+
 // SetDrives will set the given drives and the offset to which the stub drives
 // start.
 func (h *stubDriveHandler) SetDrives(offset int64, d []models.Drive) {
```

Comment:
This offset can be removed all together with the function provided above.

Reply:
Now offset is always zero. I will remove the parameter later.